codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerHandshaker.java

```diff
@@ -75,12 +75,11 @@
      *            sent to this URL.
      * @param subprotocols
      *            CSV of supported protocols. Null if sub protocols not supported.
-     * @param maxFramePayloadLength
-     *            Maximum length of a frame's payload
+     * @param decoderConfig
+     *            Frames decoder configuration.
      */
     protected WebSocketServerHandshaker(
-            WebSocketVersion version, String uri, String subprotocols,
-            int maxFramePayloadLength) {
+            WebSocketVersion version, String uri, String subprotocols, WebSocketDecoderConfig decoderConfig) {
```

Comment:
we need to preserve the old constructor as otherwise it would be a breaking change.

Reply:
reverted both constructor and 'maxFramePayloadLength ' method